client/api.ts

```diff
@@ -52,7 +52,7 @@ export default class API {
   }
 
   static getInfo(packageString: string) {
-    return API.get(`/api/size?package=${packageString}&record=true`)
+    return API.get<any>(`/api/size?package=${packageString}&record=true`)
```

Comment:
We should type this `any` — share types between the BE and FE.

Reply:
I completely agree, and we'll get there eventually. 

Ideally this should be typed as `Awaited<ReturnType<typeof getPackageStats>>`, but TS isn't happy about it:


> ./pages/scan-results/ScanResults.tsx:189:31
Type error: Property 'version' does not exist on type '{ size: number; gzip: number; parse: { baseParseTime: number; scriptParseTime: number; } | { baseParseTime?: undefined; scriptParseTime?: undefined; } | null; dependencySizes?: { name: string; approximateSize: number; }[] | undefined; ... 6 more ...; peerDependencies?: string[] | undefined; } | { ...; }'.
>  Property 'version' does not exist on type '{ size: number; gzip: number; parse: { baseParseTime: number; scriptParseTime: number; } | { baseParseTime?: undefined; scriptParseTime?: undefined; } | null; dependencySizes?: { name: string; approximateSize: number; }[] | undefined; ... 6 more ...; peerDependencies?: string[] | undefined; }'.

Do you think you can add clear typings to `package-build-stats` package?

Reply:
`/api/size` might not return `typeof getPackageStats` directly, it augments the response IIRC. 
Also, `package-build-stats` should have types, but in case you need something specific here, do you mind sending a PR there to add an export? Thanks!

Reply:
I see. I guess I underestimated the scope of this PR. Closing.